src/components/Navbar/ResetDrop.tsx

```diff
@@ -6,9 +6,9 @@ import { ThemeProvider } from '@emotion/react';
 import ResetGame from './ResetGame';
 import { useSelector } from 'react-redux';
 import { RootState } from '../../store';
-
+import MakeSpectator from '../RoomView/MakeSpectator';
 export default function BasicPopover() {
```

Comment:
We should call this ResetDrop instead of BasicPopover - just make sure to change where it's called, too